### PR TITLE
fix the typo of serviceaccount

### PR DESCRIPTION
**Pull request by jianglingxia**
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> For 1.8 Features: set Milestone to `1.8` and Base Branch to `release-1.8`
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>
> NOTE: Please check the “Allow edits from maintainers” box (see image below) to
> [allow reviewers to fix problems](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) on your patch and speed up the review process.
>
> Please delete this note before submitting the pull request.

Allow edits from maintainers checkbox
image

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/5533)

### CODE REVIEW

File: docs/tasks/configure-pod-container/configure-service-account.md [hunk 1]
```diff
@@ -146,18 +146,19 @@ Any tokens for non-existent service accounts will be cleaned up by the token con
...
 ====
-ca.crt: 1220 bytes
-token: ...
-namespace: 7 bytes
+ca.crt:         1338 bytes
+namespace:      7 bytes
+token:          eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJrdWJlcm5ldGVzL3NlcnZpY2VhY2NvdW50Iiwia3ViZXJuZXRlcy5pby9zZXJ2aWNlYWNjb3VudC9uYW1lc3BhY2UiOiJkZWZhdWx0Iiwia3ViZXJuZXRlcy5pby9zZXJ2aWNlYWNjb3VudC9zZWNyZXQubmFtZSI6ImJ1aWxkLXJvYm90LXNlY3JldCIsImt1YmVybmV0ZXMuaW8vc2VydmljZWFjY291bnQvc2VydmljZS1hY2NvdW50Lm5hbWUiOiJidWlsZC1yb2JvdCIsImt1YmVybmV0ZXMuaW8vc2VydmljZWFjY291bnQvc2VydmljZS1hY2NvdW50LnVpZCI6ImRhNjhmOWM2LTlkMjYtMTFlNy1iODRlLTAwMmRjNTI4MDBkYSIsInN1YiI6InN5c3RlbTpzZXJ2aWNlYWNjb3VudDpkZWZhdWx0OmJ1aWxkLXJvYm90In0.l5LmE4XfU_aMXr0tewL2TCfJa4-jjng9-l5yohyoBqUzt9IT45dal05q-9dJzT1oCfJTPBO8gg-6Gg-PsKv8hoTlHY5bxE1MFkVxulhns135TmUK1-PCPPT0-7okbPn6bRD0f-L0p6G4wNkUXLk3HlWM5MO2GthENopFwPvdWe5W2IbJSHVNbgbAkuC1zDapxI5uNwfdQ3oS16Tw0O1H7G4vYL2XUYgBt25kkIvvgYjLjTDwV_IC8b_zV19mXEc8Q62eg8PQAGbmm9e5mTCprNxvFF45rIET0njfiQvq8xPaYClWrpKiaSNhth_OCJqMrzJBoLDWW-orianwZ3coQg
 ```
 
 **Note:** The content of `token` is elided here.
```
Comment (xiangpengzhao) [inline docs/tasks/configure-pod-container/configure-service-account.md]: Do we really need to list the value of a token in the example? If so, this line should be removed.

Comment (Bradamant3) [reply docs/tasks/configure-pod-container/configure-service-account.md]: recommend removing token value and restoring original elided content

Comment (zhangxiaoyu-zidif) [reply docs/tasks/configure-pod-container/configure-service-account.md]: use "..." to replace token value

Comment (jianglingxia) [reply docs/tasks/configure-pod-container/configure-service-account.md]: I want to know that namespace‘s Order is a problem? or leave it alone?

Comment (xiangpengzhao) [reply docs/tasks/configure-pod-container/configure-service-account.md]: It doesn't matter. Leave the `namespace` order as you modified is ok. What you need to do now is to revert the token content :)

**Comment by jianglingxia** [comment]
I have updated it that revert change to token field,thanks!

**Comment by jianglingxia** [comment]
cc @Bradamant3

**Pull request merged**